ThatValidator.js

```diff
@@ -362,10 +347,10 @@
 
                 for(var i = 0; i < elements.length; i++)
                 {
-                    var field = elements[i];
+                    var field2 = elements[i];
 
                     //if the element has not already been found
-                    if(self.fields.indexOf(field) == -1)
+                    if(self.fields.indexOf(field2) == -1)
```

Comment:
It's a small thing. "field" is defined in the function arguments, and 
then you define it again with "var field = self.fields[i];". Very 
strictly speaking this isn't allowed.

On 9/14/2015 3:39 PM, Jonathan Dumaine wrote:
>
> In ThatValidator.js 
> <https://github.com/JonDum/ThatValidator/pull/2#discussion_r39437341>:
>
> >
> >                      //if the element has not already been found
> > -                    if(self.fields.indexOf(field) == -1)
> > +                    if(self.fields.indexOf(field2) == -1)
>
> Not sure about this. Where is the conflict? Where is |field| even 
> defined now to add the handlers?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/JonDum/ThatValidator/pull/2/files#r39437341>.
>